src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2800,7 +2800,7 @@ private void GenerateHelperFunctions(TextWriter writer)
 
```

Comment:
@vexx32, your last commit had 5 failures in `PowerShell-CI-linux`
Basic SysLog tests on Linux.Verifies logging level filtering works
```powershell
Expected $null, but got PSLogItem.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/Logging.Tests.ps1: line 256
256:         $items | Should -Be $null
```
Restart-Computer.Should restart the local computer
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Restart-Computer.Tests.ps1: line 28
28:             Restart-Computer -ErrorAction Stop | Should -BeNullOrEmpty

```
Restart-Computer.Restart-Computer Error Conditions.Should return the proper error when it occurs
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Restart-Computer.Tests.ps1: line 75
75:                 Restart-Computer -ErrorVariable RestartError 2> $null

```
Stop-Computer.Should stop the local computer
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Stop-Computer.Tests.ps1: line 29
29:             Stop-Computer -ErrorAction Stop | Should -BeNullOrEmpty

```
Stop-Computer.Stop-Computer Error Conditions.Should return the proper error when it occurs
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Stop-Computer.Tests.ps1: line 48
48:                 Stop-Computer -ErrorVariable StopError 2> $null

```